Makefile

```diff
@@ -60,7 +60,7 @@ local_test: generate
 	go test $(TEST_PARAM) ./...
 
 run: clean generate
-	$(DOCKER_COMPOSE) up --build --detach tigris_server
+	$(DOCKER_COMPOSE) up --build --detach tigris_server2
```

Comment:
Should we rename these instances instead of using a generic numeric index? Is it possible to name them in a way that reflects their purpose?

Reply:
This just second instance of tigris_server (tigris_server too :)
It's not forwarder specific it can be used for any kind of distributed testing.